frontend/package.json

```diff
@@ -10,9 +10,13 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
```

Comment:
As of late 2021, [styled-components](https://github.com/styled-components/styled-components) is not compatible with server-rendered Material UI projects. This is because babel-plugin-styled-components isn't able to work with the styled() utility inside @mui packages. See [this GitHub issue](https://github.com/mui/material-ui/issues/29742) for more details.

We strongly recommend using Emotion for SSR projects.

Reply:
Hallo Marcell, ich habe mir gestern Mui ansgechaut und installiert, denn ich habe mir gedacht, da ich kein css kann, das mir eine gute Vorlage liefern würde... ehrlicherweise, bin ich von deinem Beispiel inspiriert worden. Ist das keine so gute Idee? Viele Grüße, Marianna

Reply:
Heyho, na klar kannst du MUI nutzen, aber du hast styled components installiert, dass brauchst du nicht, da es veraltet ist und zu problemen führen kann und bisher, zumindest in diesem Pull Request nutzt du kein MUI

Reply:
Verstehe, danke für den Hinweis. Wie kann ich das rückgängig machen: reicht es wenn ich Zeile 14 aus der package.json lösche?

Reply:
genau!:)